vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/search/AbstractElementSearchAction.java

```diff
@@ -139,11 +139,16 @@ private boolean isElementVisible(WebElement element, boolean scrolled)
 
     protected static By generateCaseInsensitiveLocator(String text, String... tagNames)
     {
-        StringBuilder locator = new StringBuilder();
+        StringBuilder locator = new StringBuilder(INITIAL_SB_SIZE);
```

Comment:
https://openjdk.java.net/jeps/280 use concatenation?

Reply:
1) I recommend to remove `INITIAL_SB_SIZE`
2) JEP 280 - I would make it as a separate commit